<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>warninglistdlg</class>
 <widget class="QDialog" name="warninglistdlg">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Warning!</string>
  </property>
  <widget class="QPlainTextEdit" name="edtWarning">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>621</width>
     <height>71</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Warning related to proposed action.</string>
   </property>
   <property name="toolTipDuration">
    <number>3</number>
   </property>
   <property name="whatsThis">
    <string>Warning related to proposed action.</string>
   </property>
   <property name="placeholderText">
    <string>Warninig missing!</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listAffected">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>621</width>
     <height>321</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>List of items that will be affected by this action.</string>
   </property>
   <property name="whatsThis">
    <string>List of items that will be affected by this action.</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnProceed">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Proceed</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
